core/reactor_common.c

```diff
@@ -1101,7 +1102,20 @@ int _lf_schedule_at_tag(trigger_t* trigger, tag_t tag, lf_token_t* token) {
  * @return A handle to the event, or 0 if no new event was scheduled, or -1 for error.
  */
 trigger_handle_t _lf_schedule(trigger_t* trigger, interval_t extra_delay, lf_token_t* token) {
-    if (_lf_is_tag_after_stop_tag(current_tag)) {
+    
+    // Find the containing reactor
+    // FIXME: Is this safe? Will all the triggered reactions always be in the same containing reactor?
+    // FIXME: Is there a scenario where we want to use the local time at the "caller" rather than the reactor containing the action? 
+    self_base_t * reactor = NULL;
+    if (trigger->number_of_reactions) {
+        reactor = (self_base_t *) trigger->reactions[0]->self;
+    } else {
+        // FIXME: Is this a possibility?
```

Comment:
I suspect this is a possibility, yes. Consider a base class reactor that schedules an action but does not define any reactions to that action. It could be subclassed still without any such reactions.